system_status/requirements_for_system_status.txt

```diff
@@ -1,6 +1,5 @@
 requests==2.31.0
 SQLAlchemy==1.4.49
-psycopg2-binary==2.9.7
```

Comment:
Had wheel failures when trying to install this, but the docker image builds without it and the tests pass, so 🤷

Reply:
It would be good to have confirmation from the App team on that one.

Reply:
yeah... running locally and it is unhappy
```
python system_status.py
INFO:root:Starting system-status lambda
INFO:root:Received event: {}
ERROR:root:System-status lambda (db connection): unknown error: No module named 'psycopg2'
```
Will investigate further....